### PR TITLE
Merge contrib-also-readme

### PR DESCRIPTION
**Pull request by lpulley**
This should keep a list of contributors in README.md as well as CONTRIBUTORS.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Hyper-Orama
 [![All Contributors](https://flat.badgen.net/github/contributors/cwlowder/hyper-orama)](#contributors)
 
-A screen recorder for the hyper terminal. It publishes all captured sessions to [now.sh](https://zeit.co/now)
+A screen recorder for the hyper terminal. It publishes all captured sessions to [now.sh](https://zeit.co/now).
```
Comment (cwlowder) [inline README.md]: Perhaps we should also mention that it saves videos locally as well?

Comment (lpulley) [reply README.md]: Good call! Can you add that?

Comment (cwlowder) [reply README.md]: You can add it yourself directly in the PR! hit ctrl+g inside a reply to this thread

Comment (lpulley) [reply README.md]:
```suggestion
A screen recorder for the hyper terminal. It publishes all captured sessions to [now.sh](https://zeit.co/now) (or locally).
```

**Pull request merged**